test/functionalTest/cases/0000_version_operation/version_via_rest.test

```diff
@@ -30,7 +30,7 @@ brokerStart CB
 --SHELL--
```

Comment:
I think there are more .test using the `GET /version` appart from this one... Have you done a full pass to functional tests? Probably changes in more .test are needed.

Reply:
However, don't implemented the changes yet. Let's solve first https://github.com/telefonicaid/fiware-orion/pull/2971#discussion_r138421576

Reply:
Fixed in e88551b